src/elements/form/form.shared-is-test.css

```diff
@@ -0,0 +1,205 @@
+	--mark-text: #2c3d49;
+	--muted-background: #edf0f3;
+	--icon-valid: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none' stroke='rgba(40, 138, 106, 0.999)' stroke-width='3' stroke-linecap='round' stroke-linejoin='round'%3E%3Cpolyline points='20 6 9 17 4 12'%3E%3C/polyline%3E%3C/svg%3E");
+	--icon-invalid: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none' stroke='rgba(185, 70, 70, 0.999)' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'%3E%3Ccircle cx='12' cy='12' r='10'%3E%3C/circle%3E%3Cline x1='12' y1='8' x2='12' y2='12'%3E%3C/line%3E%3Cline x1='12' y1='16' x2='12.01' y2='16'%3E%3C/line%3E%3C/svg%3E");
+	--icon-chevron: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' viewBox='0 0 24 24' fill='none' stroke='rgba(115, 130, 140, 0.999)' opacity='0.66' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'%3E%3Cpolyline points='6 9 12 15 18 9'%3E%3C/polyline%3E%3C/svg%3E");
+	--spacing-typography: 1.5rem;
+	--form-element-outline-width: 3px;
+	--primary-focus: rgba(16, 149, 193, 0.125);
+	--input-focus: var(--primary-focus);
+}
+
+button,
+input,
+optgroup,
+select,
+textarea {
+	font-family: inherit; /* 1 */
+	font-size: 1rem; /* 1 */
+	line-height: var(--line-height); /* 1 */
+	margin: 0; /* 2 */
+}
+
+select,
+textarea,
+form {
+	display: block;
+	width: 100%;
+}
+
+input:not([type='checkbox']):not([type='radio']):not([type='range']):not([type='file']),
+select,
+textarea {
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	appearance: none;
+	padding: var(--form-element-spacing-vertical) var(--form-element-spacing-horizontal);
+	vertical-align: middle;
+}
+
+/* Remove the inheritance of text transform in Edge, Firefox, and IE */
+select {
+	text-transform: none;
+}
+
+input,
+select,
+textarea {
+	border: var(--form-element-border-width) solid var(--input-border);
+	border-radius: var(--block-round);
+	outline: none;
+	background-color: var(--input-background);
+	color: var(--text);
+	font-weight: var(--form-element-weight);
+}
+
+input::placeholder,
+select::placeholder,
+textarea::placeholder,
+input::-webkit-input-placeholder,
+select::-webkit-input-placeholder,
+textarea::-webkit-input-placeholder {
+	color: var(--muted-text);
+	opacity: 1;
+}
+
+input:active,
+select:active,
+textarea:active,
+input:focus,
+select:focus,
+textarea:focus {
+	border-color: var(--input-hover-border);
+	background-color: var(--input-hover-background);
+}
+
+input[readonly],
+select[readonly],
+textarea[readonly],
+input[disabled],
+select[disabled],
+textarea[disabled] {
+	border-color: var(--muted-border);
+	box-shadow: none;
+}
+
+input[readonly] ~ label,
+select[readonly] ~ label,
+textarea[readonly] ~ label,
+input[disabled] ~ label,
+select[disabled] ~ label,
+textarea[disabled] ~ label {
+	color: var(--muted-text);
+}
```

Comment:
This only works for

```html
<input .../>
<label>...<label>
```

We will also need


```html
<label>...<label>
<input .../>
```

and


```html
<label>...<input .../><label>
<label><input .../>...<label>
```

Reply:
Yeah, that was just a test. Thanks for the heads up